cli/cmd/diagnostics.go

```diff
@@ -36,7 +36,6 @@ This command provides subcommands to diagnose the functionality of Linkerd.`,
 	diagnosticsCmd.AddCommand(newCmdControllerMetrics())
 	diagnosticsCmd.AddCommand(newCmdEndpoints())
 	diagnosticsCmd.AddCommand(newCmdMetrics())
-	diagnosticsCmd.AddCommand(newCmdInstallSP())
```

Comment:
Now that we are removing this command, we also need to update the instruction in the help text to remove this part
```
...
  # Install service profiles for the control-plane components.
  linkerd diagnostics install-sp
```